test/jdk/java/awt/Focus/UnaccessibleChoice/AccessibleChoiceTest.java

```diff
@@ -117,12 +114,18 @@ public void start () throws IOException {
             throw new RuntimeException("Button doesn't have focus");
         }
 
+        robot.delay(500);
+
         // Press Tab key to move focus to Choice
         robot.keyPress(KeyEvent.VK_TAB);
         robot.keyRelease(KeyEvent.VK_TAB);
 
         robot.delay(500);
 
+        if (!choice.isFocusOwner()) {
+            throw new RuntimeException("Choice doesn't have focus");
+        }
+
         // Press Down key to select next item in the choice(Motif 2.1)
```

Comment:
Motif is deprecated.. can be removed
```suggestion
        // Press Down key to select next item in the choice
```

Reply:
Done